manager/integration/tests/test_scheduling.py

```diff
@@ -1722,3 +1722,67 @@ def finalizer():
 
     create_and_wait_statefulset(statefulset)
     wait_for_statefulset_pods_healthy(statefulset)
+
+
+@pytest.mark.skip(reason="TODO")
+def test_allow_empty_node_selector_volume_setting(): # NOQA
+    """
+    Test the global setting allow-empty-node-selector-volume
+
+    If true, a replica of the volume without node selector
+    can be scheduled on node with tags.
+
+    If false, a replica of the volume without node selector
+    can not be scheduled on node with tags.
+
+    Setup
+    - Prepare 3 nodes
+    - Add `AVAIL` tag to nodes
+    - Set allow-empty-node-selector-volume to `false`
+
+    When
+    - Create a Volume with 3 replicas without tag
+
+    Then
+    - All replicas can not be scheduled to the nodes
+
+    When
+    - Set allow-empty-node-selector-volume to `true`
```

Comment:
Before setting `allow-empty-node-selector-volume` to `true`,  can we add one more step of removing the `AVAIL` tag from one of the node and as a result volume replica should get scheduled on that node without tag. 

Purpose of adding the test step is to ensure `allow-empty-node-selector-volume` setting is respecting tags correctly and just not blindly overruling the schedule criteria.

Reply:
Fixed, thanks!